crowd_sim/envs/crowd_sim.py

```diff
@@ -77,10 +87,39 @@ def configure(self, config):
             logging.info("Not randomize human's radius and preferred speed")
         logging.info('Training simulation: {}, test simulation: {}'.format(self.train_val_sim, self.test_sim))
         logging.info('Square width: {}, circle width: {}'.format(self.square_width, self.circle_radius))
+        logging.info("Number of static obstacles: {}".format(self.static_obstacle_num))
+        logging.info("Static obstacles diameter range: {} - {}".format(2 * self.obstacle_min_radius, 2 * self.obstacle_max_radius))
 
     def set_robot(self, robot):
         self.robot = robot
 
+    def generate_random_obstacles(self, obs_num):
+        width = self.square_width
+        height = self.square_width
+        max_radius = self.obstacle_max_radius - self.obstacle_min_radius
+        min_radius = self.obstacle_min_radius
+        self.obs = []
```

Comment:
I think we set `self.obs` for the first time here. Would be great idea to add the following line
```
self.obs = None
```
to the `__init__()` method.

Reply:
good idea